next.config.js

```diff
@@ -21,6 +21,11 @@ const config = {
         destination: "/pipelines/:path*",
         permanent: false,
       },
+      {
```

Comment:
The goal of this PR is to avoid redirect the user to the legacy version of openhexa if the workspaces feature is enabled for his account. I think this line will prevent that.